reference/catalog/brands_catalog.v3.yml

```
@@ -1587,6 +1587,142 @@ paths:
     parameters:
       - $ref: '#/components/parameters/Accept'
       - $ref: '#/components/parameters/BrandIdParam'
+  '/catalog/brands/metafields':
+    get:
+      summary: Get All Metafields
+      tags:
+        - Metafields
+      description: Get all brand metafields.
+      operationId: getAllBrandMetafields
+      responses:
+        '200':
+          description: |
+            List of `Metafield` objects.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MetaFieldCollectionResponse'
+        '500':
+          description: Internal Server Error
+      parameters:
+        - $ref: '#/components/parameters/PageParam'
+        - $ref: '#/components/parameters/LimitParam'
+        - $ref: '#/components/parameters/MetafieldKeyParam'
+        - $ref: '#/components/parameters/MetafieldKeyInParam'
+        - $ref: '#/components/parameters/MetafieldNamespaceParam'
+        - $ref: '#/components/parameters/MetafieldNamespaceInParam'
+        - $ref: '#/components/parameters/DirectionParam'
+    post:
+      summary: Create multiple Metafields
+      tags:
+        - Metafields
+      description: Create multiple metafields.
+      operationId: createBrandMetafields
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                allOf:
+                  - $ref: '#/components/schemas/MetafieldBase_Post'
+                  - type: object
+                    properties:
+                      resource_id:
+                        type: string
+                        example: "42"
+                        description: |
+                          The id for the cart with which the metafield is associated.
```

```suggestion
                          The ID for the brand with which the metafield is associated.
```
check the cut and paste. capitalize ID throughout where not in code quotes.